src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocLexer.g4

```diff
@@ -37,8 +37,7 @@ import java.util.*;
 
 }
 
-LEADING_ASTERISK : ( (' '|'\t') {_tokenStartCharPositionInLine == 0}? ) (' '|'\t')* '*'
-      | '*' {_tokenStartCharPositionInLine == 0}?
+LEADING_ASTERISK : (' '|'\t')* '*'
```

Comment:
@ps-sp Your tests also fail to display any line/column numbers as I can't find any of your print out numbers in it.

> Should we create an issue for ANTLR project ?

Yes, please create an issue with them.